test/integration/default/serverspec/server_spec.rb

```diff
@@ -0,0 +1,12 @@
+require 'serverspec'
+
+set :backend, :exec
+
+describe service('varnishlog') do
```

Comment:
I don't see the point in adding tests for the actual cookbook. I would rather open a pr on the actual cookbook and add tests.

I can add the firewall recipes.